GenumKit/Parsers/FontsFileParser.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  FontsFileParser.swift
+//  Pods
```

Comment:
Hey, it's the professional nitpicker again! :stuck_out_tongue_winking_eye: 

Please make the file headers for your new files conform to the other ones, especially mention the MIT License, so something like:

```
//
// GenumKit
// File created by Derek Ostrander on 3/7/16.
// Copyright (c) 2015 Olivier Halligon
// MIT Licence
//
```